kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/assertj/ConditionAssert.java

```diff
@@ -33,53 +35,93 @@ public AbstractLongAssert<?> observedGeneration() {
         return Assertions.assertThat(actual.getObservedGeneration());
     }
 
+    public ConditionAssert hasObservedGeneration(Long expected) {
```

Comment:
I realised that it's slightly nicer to provide `hasXX()` methods on asserts, as well as the `xx()` methods, to make it easier to chain assertions.

Reply:
When/if we merge this I'll fix up the `ProxyReconcilerTest` to use these ones too.